software/messaging/src/test/java/brooklyn/entity/messaging/kafka/KafkaIntegrationTest.java

```
@@ -128,8 +128,13 @@ public Void call() {
         KafkaSupport support = new KafkaSupport(cluster);
 
         support.sendMessage("brooklyn", "TEST_MESSAGE");
-        String message = support.getMessage("brooklyn");
-        assertEquals(message, "TEST_MESSAGE");
+        try {
+            Thread.sleep(Duration.seconds(5).toMilliseconds());
```

Probably should let this propagate with `Time.sleep()` since it's in a test?